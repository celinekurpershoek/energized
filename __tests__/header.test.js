import React from "react";
import renderer from "react-test-renderer";
import Header from "../src/components/header";

it("Correctly render header component", () => {
  const component = renderer.create(<Header title="Hello test" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
