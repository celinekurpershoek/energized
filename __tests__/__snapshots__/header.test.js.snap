// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly render header component 1`] = `
<header
  className="app-header"
>
  <h1>
    Hello test
  </h1>
</header>
`;
